### DISCUSSION TITLE
windows NSIS uninstaller issue with tauri-cli 1.4.0

### DISCUSSION THREAD

**Discussion by YichenZhou113** [tauri-apps/tauri#7313] [Q&A] [answered]
Seems that with the latest tauri-cli 1.4.0, the NSIS generated `.exe` installer on windows will have issue when trying to uninstall the existing application. 
Though the uninstall was actually completed and successful, it will still pop up a window and say "Unable to uninstall".
<img width="300" alt="Screenshot 2023-06-28 at 12 17 28 PM" src="https://github.com/tauri-apps/tauri/assets/40067411/1b923c04-53f0-4401-b7dc-9ccdf270edfb">
Switching back to tauri-cli 1.3.0 will fix the issue.

**Comment by FabianLars** [comment]
Please open an issue (bug report) about this instead of a discussion. thank you.

**Reply by FabianLars** [reply, marked as answer]
#7317
